### ISSUE TITLE
Pop does not detect Windows until I install manually os-prober

### ISSUE THREAD

**Issue by All3xJ** [pop-os/pop#1248]
**Distribution (run `cat /etc/os-release`):**
```
NAME="Pop!_OS"
VERSION="20.04 LTS"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 20.04 LTS"
VERSION_ID="20.04"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal
LOGO=distributor-logo-pop-os
```
 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**
os-prober and GRUB

**Issue/Bug Description:**
I installed PopOS in my sister computer (Acer Aspire 5750G). I was really surprised that I had to MANUALLY install os-prober and then manually update grub to detect Windows and to show the boot choice between Pop and Windows.

**Steps to reproduce (if you know):**
I have this problem with Acer Aspire 5750G, in which GRUB was installed by Pop installer, not systemd-boot.
So basically windows could never be detected, because os-prober was not installed.

**Expected behavior:**
A new Linux user should expect to choose if run Linux or Windows at the boot.

**Other Notes:**
Why don't PopOS include by default os-prober?

It's a real shame for the new Linux users that could be in difficulty.

**Comment by FDiskas** [comment]
Same for me. I got multiboot with: deepinos, elementary os, windows, fedora. And after upgrading popOS to latest 20.10 all entries was gone. And I was unable to fix it using some tools. Only after reinstalling "deepin os" I got again nice looking grub with all operating systems available.
